api/app/Jobs/HandleMergeWebhookJob.php

```diff
@@ -26,16 +28,28 @@ public function handle()
   {
     $payload = $this->webhookCall->payload();
     $payload = json_decode($payload['payload']);
+    $message = [
+      'type' => 'merge',
+      'name' => $payload->pull_request->merged_by->login,
+      'avatar' => $payload->pull_request->merged_by->avatar_url,
+      'pr_title' => $payload->pull_request->title,
+      'pr_url' => $payload->pull_request->html_url
+    ];
     if ($payload->action === 'closed' && $payload->pull_request->merged === true) {
```

Comment:
Create an enum for `closed` instead of hard coding it. 

```suggestion
    if ($payload->action === PRStatusEnum::CLOSED && $payload->pull_request->merged) {
```